grails-app/controllers/org/broadinstitute/orsp/api/NewConsentGroupController.groovy

```diff
@@ -127,4 +127,36 @@ class NewConsentGroupController extends AuthenticatedController {
         response.status = 200
         render([consent: consent] as JSON)
     }
+
+    Collection<ConsentCollectionLink> getConsentCollectionLinks() {
+        Issue issue = queryService.findByKey(params.consentKey)
+        Collection<ConsentCollectionLink> collectionLinks = queryService.findCollectionLinksByConsentKey(issue.projectKey)
+        render([collectionLinks: collectionLinks.linkedProject] as JSON)
+        collectionLinks
+    }
+
+    /**
+     * This action breaks the link between a project and the consent from the point of view of the consent.
+     *
+     */
+    def unlinkAssociatedProjects () {
+        Issue issue = queryService.findByKey(params.consentKey)
+        Issue target = queryService.findByKey(request.JSON["projectKey"] as String)
+        def links = ConsentCollectionLink.findAllByProjectKeyAndConsentKey(target.projectKey, issue.projectKey)
+        deleteCollectionLinks(links)
+        response.status = 200
```

Comment:
Note that this will always be a 200 even if the delete fails. It might make more sense return a boolean or exception from the delete call and then use that to craft the response status.

Reply:
You're right, if any error occurs now it will return a 500